core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -121,6 +120,239 @@ export class ContributorAdminStatsTable implements OnInit {
     }
   }
 
+  pluralise(num: number, word: string): string {
+    return num === 1 ? word : word + 's';
+  }
+
+  getCountDisplay(
```

Comment:
Not sure what this means. Perhaps a name like getTextForColumnHeader() or something simlar might be better (I'm not sure this is correct since I don't know what this is for).

Reply:
Done